### PR TITLE
feat: add track event to the CLI

### PR DESCRIPTION
**Pull request by Kubessandra**
Add a method to track event from the CLI

### CODE REVIEW

File: packages/anima-cli/src/commands/sync.ts [hunk 1]
```diff
@@ -166,6 +169,17 @@ export const handler = async (_argv: Arguments): Promise<void> => {
...
   spanGetDSToken.finish();
 
+  trackEvent([
+    {
+      action: 'anima-cli.sync.started',
+      time: Date.now(),
+      eventParams: {
+        platform: 'anima-cli',
+        utm_source: 'anima-cli',
+        team_id: teamId,
```
Comment (Haimonline) [inline packages/anima-cli/src/commands/sync.ts:179]: Is the team_id reported on the rest of the sync-related operations in the backend? so we'll be able to track a flow e2e

**Review by Haimonline — APPROVED** [review]
LGTM

File: packages/anima-cli/src/commands/sync.ts (hunk 1 above)
Comment (Kubessandra) [reply packages/anima-cli/src/commands/sync.ts:180]: Yes, we have access to.

We already have all the flow with sentry tracing normally

**Comment by Kubessandra** [comment]
> LGTM, left a comment

Actually not working in mixpanel trying to figure out why

**Review by Datlyfe — APPROVED** [review]

**Pull request merged**